lib/run/index.js

```diff
@@ -173,7 +177,7 @@ module.exports = function (options, callback) {
             fileResolver: new SecureFS(options.workingDir, options.insecureFileRead),
             requester: {
                 useWhatWGUrlParser: true,
-                cookieJar: request.jar(),
+                cookieJar: cookieJar,
```

Comment:
```suggestion
                cookieJar: options.cookieJar || request.jar(),
```

Reply:
I have done this already when declaring the variable above this

Reply:
And I need to do that because that variable is being used for export too below